package/python-kiwi-spec-template

```diff
@@ -231,14 +273,13 @@ outside of the scope of kiwi appliance building.
 %ifarch %ix86 x86_64
 %package -n kiwi-pxeboot
 Summary:        KIWI - PXE boot structure
+Group:          System/Management
```

Comment:
I had copied this from the old kiwi package, which has different Group entries for the main package and some of the subpackages.